packages/node-bridge/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "allowJs": true,
     "checkJs": true,
-    "lib": ["ES2020"],
+    "lib": ["ES2020", "DOM"],
```

Comment:
**question (non-blocking):** The "DOM" lib here will make TS believe it's OK to use all DOM APIs, but Edge Functions don't support all of them. Could this become an issue?